### PR TITLE
BR/Black Awareness Day as National Holiday

### PR DESCRIPTION
**Pull request by tiagoornelas**
Updates Black Awareness Day for BR since it is a national holiday beginning on 2024.

News about the law that changed to National Holiday: https://g1.globo.com/politica/noticia/2023/12/21/lula-sanciona-lei-que-torna-nacional-o-feriado-da-consciencia-negra-em-20-de-novembro.ghtml

Wikipedia: https://pt.wikipedia.org/wiki/Dia_Nacional_de_Zumbi_e_da_Consci%C3%AAncia_Negra

The Wikipedia article was also updated to highlight the change to national holiday:

> Foi oficializado feriado nacional em 21 de dezembro de [2023](https://pt.wikipedia.org/wiki/2023).[[3]](https://pt.wikipedia.org/wiki/Dia_Nacional_de_Zumbi_e_da_Consci%C3%AAncia_Negra#cite_note-lei-3)

### CODE REVIEW

**Comment by tiagoornelas** [comment]
Hello, @commenthol . How are you doing?

I hope the changes in this PR is aligned with the best practices of your project. It will be really useful to update this specific holiday into national holiday for the brazilian users.

I'll be waiting for your review. Wish you the best!

**Comment by commenthol** [comment]
Hi @tiagoornelas,
Thanks for the PR.
It looks that the fixtures are not up to date. Can you please run and check the changes?
```sh
npx mocha test/all.mocha.js --writetests --countries BR
```

**Comment by tiagoornelas** [comment]
> Hi @tiagoornelas, Thanks for the PR. It looks that the fixtures are not up to date. Can you please run and check the changes?
> 
> ```shell
> npx mocha test/all.mocha.js --writetests --countries BR
> ```

Hello, @commenthol !

I ran the command and all of tests are passing locally without any changes on code. Looking further, I noticed that the CI's test expects the holiday for the **states and cities on 2024**. Since I added the "active.to" parameter to the states and cities' holidays is it possible that the expectation itself is mistaken?

Since the holiday, from Dec 2023, turned into a national holiday, should it be expected only for the BR country and no longer for the states and cities?

Please check if this makes sense. Or, if doesn't, could you share your ideas on how to solve this? Thank you!

**Comment by NatanaelHZ** [comment]
Hello,

I would like to express my support for this pull request. As a package user, I confirm the importance of adding Black Awareness Day as a national holiday starting in 2024.

The Law sanctioned on December 21, 2023, made November 20th a national holiday, as widely reported. More details can be found on the official Brazilian government website: https://www.planalto.gov.br/ccivil_03/_ato2023-2026/2023/lei/l14759.htm

I look forward to the resolution of this update.

Thank you for your dedication to keeping the package up to date!

File: data/countries/BR.yaml [hunk 1]
```diff
@@ -78,6 +78,12 @@ holidays:
         name:
           pt: Proclamação da República
           en: Republic Day
+      11-20:
+        name:
+          pt: Dia da Consciência Negra
+          en: Black Awareness Day
+        active:
+              - from: 2023-12-21
```
Comment (commenthol) [inline data/countries/BR.yaml:86]: @tiagoornelas 
The indenting is incorrect here. Please use
```yaml
        active:
          - from: 2023-12-21
```

**Comment by commenthol** [comment]
Hi @tiagoornelas, 
Can you please follow the instructions in CONTRIBUTING.md?
The important step is to run `npm run yaml` to convert your changes to json/ code.
This will solve passing the tests

**Comment by tiagoornelas** [comment]
Hello, @commenthol !

Can you approve the workflow to check if the new commit solves the issues addressed previously?

Happy holidays!

**Pull request merged**